core/storage/learner_group/gae_models.py

```diff
@@ -308,3 +308,48 @@ def get_by_facilitator_id(
         )).fetch()
 
         return found_models
+
+    @classmethod
+    def get_by_learner_user_id(
+        cls, learner_user_id: str
+    ) -> Sequence[LearnerGroupModel]:
+        """Returns a list of all LearnerGroupModels that have the given
+        learner user id.
+
+        Args:
+            learner_user_id: str. The id of the learner.
+
+        Returns:
+            list(LearnerGroupModel)|None. A list of all LearnerGroupModels that
+            the given learner is part of or None if no such learner group
+            models exist.
+        """
+        found_models: Sequence[LearnerGroupModel] = cls.get_all().filter(
+            datastore_services.any_of(
+                cls.learner_user_ids == learner_user_id
+        )).fetch()
+
+        return found_models
+
+    @classmethod
+    def get_by_invited_learner_user_id(
+        cls, invited_learner_user_id: str
+    ) -> Sequence[LearnerGroupModel]:
+        """Returns a list of all LearnerGroupModels that have the given
```

Comment:
This docstring is exactly the same as the previous function's. So why have two functions for the same thing?

Reply:
There was a problem with this docstring, fixed it. Thanks!